commcare_connect/opportunity/models.py

```diff
@@ -70,3 +70,70 @@ class DeliverForm(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class LearnModule(models.Model):
+    app = models.ForeignKey(
+        CommCareApp,
+        on_delete=models.CASCADE,
+        related_name="learn_module",
+        related_query_name="learn_module",
```

Comment:
These should be plural, also if they are the same you only need to specify one:
```suggestion
        related_name="learn_modules",
```